### ISSUE TITLE
As a consumer, I want to view a "value snapshot" of each of the schools listed in my search results so that at a glance I can determine which schools I want to investigate further

### ISSUE THREAD

**Issue by jjoteal** [18F/college-choice#190]
This issue relates to the display of the top-level charts (average net cost, graduation rate, and salary) in the search results, as on the school pages.

**Comment by jjoteal** [comment]
Mobile reference comp:
![image](https://cloud.githubusercontent.com/assets/12192706/8782948/c9a1e3dc-2ee6-11e5-96c9-7f7522ac2a6d.png)

Desktop reference comp:
![image](https://cloud.githubusercontent.com/assets/12192706/8782958/d8ff54ea-2ee6-11e5-8d79-bb39cf140c3e.png)

**Comment by shawnbot** [comment]
Note that the implementation of the salary charts is covered in #180, which, as of this moment, is blocked by a lack of data. So this is blocked by that issue and should be deferred until #152 is merged.

**Comment by shawnbot** [comment]
Assigning to @meiqimichelle for HTML & CSS work, then I will follow up with data.

**Comment by meiqimichelle** [comment]
@shawnbot #206 should cover the HTML/CSS. Sending this back to you for teh dataz.

**Comment by shawnbot** [comment]
@meiqimichelle I think the only thing missing here is the little legend bit. Do you have time for that?

**Comment by shawnbot** [comment]
This is 95% done, we just need the little legend in there to call it 100%. Closing in favor of a more specific issue.

**Comment by shawnbot** [comment]
Whoops, reopening for acceptance.

**Comment by ultrasaurus** [comment]
Note: on some desktops sizes there a visual bug of text overlapping:  https://github.com/18F/college-choice/issues/225

**Comment by ErieMeyer** [comment]
LGTM!
